Dockerfile

```diff
@@ -1,54 +1,32 @@
-FROM mono:6.0
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1
```

Comment:
Any specific reason for mcr.microsoft?

Reply:
It's the official dotnet core image?
[https://hub.docker.com/_/microsoft-dotnet-core-sdk/](url)